README.md

```diff
@@ -2,18 +2,19 @@
 
-The identity ecosystems that support these use cases are individually and collectively vast, including classical forms of identities such as email addresses, phone numbers, and certificates, to new forms of identities built on standards such as [OAuth [1]](https://datatracker.ietf.org/doc/html/rfc6749) and [Verifiable Credentials [2]](https://www.w3.org/TR/vc-data-model/). These different identities vary in type (what sort of entity does the identity refer to?), uniqueness (does it refer to one or more entities?), security (how easy is it for identities to be spoofed?), longevity (is the identifier still valid at this moment?), and so on.
+The identity ecosystems that support these use cases are individually and collectively vast, including classical forms of identities such as email addresses, phone numbers, and certificates, to new forms of identities built on standards such as [OAuth [1]](https://datatracker.ietf.org/doc/html/rfc6749) and [Verifiable Credentials [2]](https://www.w3.org/TR/vc-data-model/). These different identities vary in type (what sort of entity does the identity refer to?), uniqueness (does it refer to one or more entities?), security (how easy is it for identities to be spoofed?), longevity (is the identifier for some identity still valid at this moment?), and so on.
 
-This program has several high-level goals. First, the program will survey the standardization landscape to determine what types of identity mechanisms exist and where they are being developed. Second, the program will examine the existing use of these identity mechanisms within the Internet, seeking to understand how identities are used or misused to solve real world problems. Third, the program will discuss what use cases are currently uncovered or otherwise inadequately addressed, with the intent of identifying gaps and opportunities for technical solutions to known problems that could be developed within the IETF or other venues. 
+This program has several high-level goals. First, the program will survey and collate a non-exhaustive list of identity mechansisms standardized or otherwise impacted by standards developed within the IETF. Second, the program will examine existing uses of these identity mechanisms within the Internet, seeking to understand how identities are used or misused to solve real world problems. Third, the program will discuss what use cases are currently uncovered or otherwise inadequately addressed, with the intent of identifying gaps and opportunities for technical solutions to known problems that could be developed within the IETF.
```

Comment:
I understand you want to narrow the scope but I'm not sure if the whole exercise of still that valuable if we only restrict ourself to the IETF. I thought the main problem is the zoo of approaches that exist in all different fora.